spec/controllers/api/v1/media_controller_spec.rb

```diff
@@ -297,6 +297,25 @@
       let(:parent) { create(:workflow, project: project) }
 
       it_behaves_like "has_many media", :workflow, :attached_images, %i(index create show destroy), 'image/jpeg'
+      it_behaves_like "has_one media", :workflow, :classifications_export, %i(index), 'text/csv'
+
+      describe "classifications_exports #index" do
+        let!(:resources) do
+          create_list :medium, 2, linked: parent, content_type: "text/csv",
```

Comment:
That's what projects were doing before I got there. The both still pass when there's only one, though, so updated.